README.md

```diff
@@ -19,13 +19,14 @@ This SDK for [Node.js](https://nodejs.org/) was created based on the [Zenvia](ht
 - [Prerequisites](#prerequisites)
 - [Installation](#installation)
 - [Basic Usage](#basic-usage)
-  - [Sending your first message](#sending-your-first-message)
+  - [Send your first message](#send-your-first-message)
+  - [Send a batch](#send-a-batch)
   - [Subscribe for messages](#subscribe-for-messages)
   - [Subscribe for message status](#subscribe-for-message-status)
-  - [Receiving message and message status events](#receiving-message-and-message-status-events)
-  - [Getting messages report](#getting-messages-report)
-  - [Getting flow report](#getting-flow-report)
-  - [Listing your templates](#listing-your-templates)
+  - [Receive message events and message status events](#receive-message-events-and-message-status-events)
+  - [Get a messages report](#get-a-messages-report)
```

Comment:
Já que você tá arrumando o inglês o certo aqui seria "message report" em vez de "messages report"

Reply:
O conteúdo do relatório é sobre uma mensagem?

"To get information on how many messages you've sent or have received during a period of time ..." (texto do "Get a messages report" do Readme)

Eu intepretei que o relatório possui conteúdoi de várias mensagens. Nesse caso, o "message report" está correto, pois seria um relatório sobre/de várias mensagens;  (I.E: relatório de mensagens).

Reply:
Tipo, normalmente palavras em posição de "adjetivo" são no singular, era só disso que eu tava falando: "message report" em vez de "messages report".

Quanto ao conteúdo do relatório, é de total de mensagens, não vem as mensagens em si, mas acho que no singular "message" fica melhor.